apps/src/music/context.ts

```diff
@@ -4,9 +4,16 @@ import MusicPlayer from './player/MusicPlayer';
 /** Provides access to the Analytics reporter object */
 export const AnalyticsContext = React.createContext(null);
 
+/** Provicess access to commonly used APIs related to the playing state. */
+export const ExecutionContext = React.createContext(null);
```

Comment:
Not blocking, but you could type this if you'd like, e.g.

```
type ExecutionInfo (// or some other name //) = {
   getIsExecutionPlay: () => boolean;
}

export const ExecutionContext<ExecutionInfo> = React.createContext<ExecutionInfo>({ getIsExecutionPlay: () => false });
```

Reply:
Will do.